store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/SourceOfTruth.kt

```diff
@@ -102,6 +102,19 @@ interface SourceOfTruth<Key : Any, Local : Any> {
             realDeleteAll = deleteAll
         )
 
+        fun <Key : Any, Local : Any, Output : Any> of(
```

Comment:
just to confirm, this is additive? I assume all past impls of source of truth are still valid?

Reply:
Removed! This method should have been removed in the refactor away from Superstore and Warehouse